tests/test_create.py

```diff
@@ -1063,6 +1063,42 @@ def side_effect(self, url, **kwargs):
                 # The mock should have been called with our local channel URL though.
                 assert result_dict.get('local_channel_seen')
 
+    def test_create_from_extracted(self):
+        # Test that we can link a package from its extracted directory
+        # even if the tarball is no longer available.
+        pkgs_dir = PackageCache.first_writable().pkgs_dir
+        mkdir_p(pkgs_dir)
+        pkgs_dir_hold = pkgs_dir + '_hold'
```

Comment:
Thanks, I'll give this a shot. I copied the test setup from another integration test, so if this works well I'll go ahead and modify other tests with a similar setup accordingly.